save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarLinks.kt

```diff
@@ -0,0 +1,127 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.frontend.components.topbar
+
+import com.saveourtool.save.utils.SAVE_CLOUD_GITHUB
+import com.saveourtool.save.validation.FrontendRoutes
+
+import csstype.ClassName
+import csstype.Width
+import csstype.rem
+import history.Location
+import js.core.jso
+import react.FC
+import react.Props
+import react.dom.aria.AriaRole
+import react.dom.aria.ariaExpanded
+import react.dom.aria.ariaLabelledBy
+import react.dom.html.ReactHTML.a
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.li
+import react.dom.html.ReactHTML.ul
+import react.router.dom.Link
+import react.router.useNavigate
+import react.useState
+
+import kotlinx.browser.window
+
+val topBarLinks = topBarLinks()
+
+/**
+ * [Props] of the top bar links component
+ */
+external interface TopBarLinksProps : Props {
+    /**
+     * Is location
```

Comment:
Please proofread the KDoc.

Reply:
Done [faec44c](https://github.com/saveourtool/save-cloud/pull/1540/commits/faec44ca7ee13b63b5e76ba2be295039cdb7c3e1)